src/Robot-Dobot_MG400/main.py

```diff
@@ -195,10 +203,21 @@ def deinitialize(self):
 
     def configure(self):
 
+        self.movement_height = float(self.movement_height)
```

Comment:
I think the conversion to float should already happen in set_guiparameter, so that self.movement_height always has the right type, even if in the future someone wants to use it already in initialize etc

Reply:
Moved it to get_GUIparameter